src/Hacks/antiaim.cpp

```diff
@@ -1,30 +1,112 @@
 #include "antiaim.h"
+#include "autowall.h"
 
-#include "aimbot.h"
-#include "../settings.h"
-#include "../Hooks/hooks.h"
-#include "../Utils/math.h"
-#include "../Utils/entity.h"
-#include "../interfaces.h"
 #include "valvedscheck.h"
 
 bool Settings::AntiAim::Yaw::enabled = false;
+bool Settings::AntiAim::FreeStanding::enabled = false;
 bool Settings::AntiAim::Pitch::enabled = false;
-
-AntiAimType_Y Settings::AntiAim::Yaw::type = AntiAimType_Y::NONE;
-AntiAimType_Y Settings::AntiAim::Yaw::typeFake = AntiAimType_Y::NONE;
+bool Settings::AntiAim::Fake::enabled = false;
+AntiAimType_Y Settings::AntiAim::Yaw::type = AntiAimType_Y::SPIN_FAST;
 AntiAimType_X Settings::AntiAim::Pitch::type = AntiAimType_X::STATIC_DOWN;
-
-bool Settings::AntiAim::HeadEdge::enabled = false;
-float Settings::AntiAim::HeadEdge::distance = 25.0f;
-
+AntiAimType_Fake Settings::AntiAim::Fake::type = AntiAimType_Fake::STATIC_LEFT;
 bool Settings::AntiAim::AutoDisable::noEnemy = false;
 bool Settings::AntiAim::AutoDisable::knifeHeld = false;
+
 bool Settings::AntiAim::LBYBreaker::enabled = false;
 float Settings::AntiAim::LBYBreaker::offset = 180.0f;
 
-float AntiAim::lastRealYaw = 0.0f;
-float AntiAim::lastFakeYaw = 0.0f;
+QAngle AntiAim::real;
+QAngle AntiAim::fake;
+
+static bool GetBestHeadAngle(QAngle& angle)
+{
+	float b, r, l;
+
+	Vector src3D, dst3D, forward, right, up, src, dst;
+
+	trace_t tr;
+	Ray_t ray, ray2, ray3, ray4, ray5;
+	CTraceFilter filter;
+
+	C_BasePlayer* localplayer = (C_BasePlayer*) entityList->GetClientEntity(engine->GetLocalPlayer());
+	if (!localplayer)
+		return false;
+
+	QAngle viewAngles;
+	engine->GetViewAngles(viewAngles);
+
+	viewAngles.x = 0;
+
+	Math::AngleVectors(viewAngles, forward, right, up);
+
+	auto GetTargetEntity = [ & ] ( void )
```

Comment:
instead of iterating these again here, you should just set it in the aimbot loop, I think there's already a variable in there I use to draw the aimspot